Compression/src/CompressDigiMCsCheck_module.cc

```diff
@@ -239,15 +258,15 @@ namespace mu2e {
 
       unsigned int n_old_calo_shower_sims = oldCaloShowerSims.size();
       for (unsigned int i_old_calo_shower_sim = 0; i_old_calo_shower_sim < n_old_calo_shower_sims; ++i_old_calo_shower_sim) {
-	unsigned int i_new_calo_shower_sim = i_old_calo_shower_sim;
-	const auto& i_oldCaloShowerSim = oldCaloShowerSims.at(i_old_calo_shower_sim);
-	const auto& i_newCaloShowerSim = newCaloShowerSims.at(i_new_calo_shower_sim);
-
-	const auto& i_oldCaloShowerStepPtr = *(i_oldCaloShowerSim.caloShowerSteps().begin());
-	const auto& i_newCaloShowerStepPtr = *(i_newCaloShowerSim.caloShowerSteps().begin());
-	if (i_oldCaloShowerStepPtr->momentumIn() != i_newCaloShowerStepPtr->momentumIn()) {
-	  throw cet::exception("CompressDigiMCsCheck") << "Old and new CaloShowerStepPtrs do not match" << std::endl;
-	}
+        unsigned int i_new_calo_shower_sim = i_old_calo_shower_sim;
+        const auto& i_oldCaloShowerSim = oldCaloShowerSims.at(i_old_calo_shower_sim);
+        const auto& i_newCaloShowerSim = newCaloShowerSims.at(i_new_calo_shower_sim);
+
+        const auto& i_oldCaloShowerStepPtr = *(i_oldCaloShowerSim.caloShowerSteps().begin());
+        const auto& i_newCaloShowerStepPtr = *(i_newCaloShowerSim.caloShowerSteps().begin());
+        if (i_oldCaloShowerStepPtr->momentumIn() != i_newCaloShowerStepPtr->momentumIn()) {
+          throw cet::exception("CompressDigiMCsCheck") << "Old and new CaloShowerStepPtrs do not match" << std::endl;
+        }
       }
     }
```

Comment:
There is no check of CaloClusterMC.  That is used in the current workflow.  I suggest to replace the CaloShowerSim check with a CaloClusterMC check.

Reply:
Added in commit 337430fac